packages/amplify-e2e-core/src/categories/storage.ts

```diff
@@ -410,7 +410,7 @@ export function addS3AndAuthWithAuthOnlyAccess(cwd: string, settings: any): Prom
       .wait('Provide a friendly name for your resource')
       .sendCarriageReturn() // Default name
       .wait('Provide bucket name')
-      .sendCarriageReturn() // Default name
+      .sendLine( settings?.bucketName || RETURN )
```

Comment:
Is `settings.bucketName` used?

Reply:
This is an option required if we were to pass a test specific bucketName. This was added to support an E2E test ( to test hyphenated names) which has been removed.

Reply:
can we keep this in , in case we require test specific bucket-names. ?

Reply:
removing this.